app/pages/sections/transport/departureTransportMeans/TransportMeansSection.scala

```diff
@@ -16,12 +16,14 @@
 
 package pages.sections.transport.departureTransportMeans
 
-import models.Index
 import pages.sections.Section
-import play.api.libs.json.{JsObject, JsPath}
+import play.api.libs.json.JsPath
+import play.api.libs.json.JsArray
 
-case class TransportMeansSection(index: Index) extends Section[JsObject] {
+case object TransportMeansSection extends Section[JsArray] {
```

Comment:
I don't actually think the type argument does anything here. Can either refactor to remove the type argument from Section and ReadOnlySection, or change this to a JsObject